frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -84,6 +84,22 @@ export async function click(elem: any, timeout?: number) {
   await elem.click();
 }
 
+export async function fillInput(elem: any, value: string) {
+  // Sometimes there seems to be an issue with clear() method not clearing the input
+  let attempts = 3;
+  do {
+    --attempts;
+    if (attempts < 0) {
+      throw Error(`Failed to fill input with value: '${value}'.`);
```

Comment:
This isn't new, but I don't think this will be ever reached because of the `attempts > 0` check in the while condition. I think we can remove `attempts > 0` below.

Reply:
good catch, fixed -thanks